src/main/java/com/google/collegeplanner/data/Class.java

```diff
@@ -0,0 +1,113 @@
+package com.google.collegeplanner.data;
```

Comment:
For the specific sections, would you want me to refer to them as "Section" or something else? Originally I had it as Course and Class, but I saw your design doc and it referred to Courses as class objects so I got a little confused.

Reply:
oh sorry. I mostly wrote the design doc before we got into the coding part. I think we should use the terms Course and Section since those are the words that are used in the rest of the code